sources/lib/components/station_ui/html/card.ex

```diff
@@ -140,29 +101,44 @@ defmodule StationUI.HTML.Card do
   defp base_horizontal_content_classes, do: @base_horizontal_content_classes
 
   attr :class, :any, default: ""
-  slot :inner_block, required: true
+  attr :direction, :string
+
   slot :header
 
-  slot :content do
+  slot :content, required: true do
     attr :class, :string
   end
 
-  def card_horizontal(assigns) do
+  def card(%{direction: "horizontal"} = assigns) do
     ~H"""
     <div class={[base_horizontal_classes() | List.wrap(@class)]}>
       <div class={base_horizontal_inner_classes()}>
         <header :for={header <- @header}>
           {render_slot(header)}
         </header>
-        <.content_card_horizontal slot={@content} />
+        <.content_card slot={@content} direction="horizontal" />
+      </div>
+    </div>
+    """
+  end
+
+  def card(assigns) do
+    ~H"""
+    <div class={[base_classes() | List.wrap(@class)]}>
+      <div class={base_inner_classes()}>
+        <header :for={header <- @header}>
+          {render_slot(header)}
+        </header>
+        <.content_card slot={@content} />
       </div>
     </div>
     """
   end
 
   attr :slot, :any, required: true
+  attr :direction, :string, default: ""
 
-  defp content_card_horizontal(assigns) do
+  defp content_card(assigns) do
```

Comment:
I could have pattern-matched the content_card function the way I did the card function. But there is only one word's difference between the two content_card function bodies, so I used one `if` statement to consolidate.